aws/data_source_aws_lambda_alias.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsLambdaAlias() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsLambdaAliasRead,
+
+		Schema: map[string]*schema.Schema{
+			"function_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"alias_name": {
```

Comment:
Since the Lambda API refers to this as `Name` and the `aws_lambda_alias` resource uses a `name` argument, we will change this attribute to match. 👍